grails-app/conf/Config.groovy

```diff
@@ -87,13 +87,16 @@ grails.hibernate.pass.readonly = false
 // configure passing read-only to OSIV session by default, requires "singleSession = false" OSIV mode
 grails.hibernate.osiv.readonly = false
 
+quartz.autoStartup = false
+
 environments {
     development {
         grails.logging.jul.usebridge = true
     }
     production {
         grails.logging.jul.usebridge = false
-        // TODO: grails.serverURL = "http://www.changeme.com"
+        quartz.autoStartup = true
+        grails.serverURL = "http://www.mama-ng-scheduler.com"
```

Comment:
I think ignore this comment for now. I'm pushing an agenda (namely, trying to make the repository itself re-usable for multiple projects, which this line would prevent) but likely figuring this out is not relevant this PR.